src/MainPage.js

```diff
@@ -43,7 +43,7 @@ const MainPage = (props) => {
             <div id="total">
                 <h4 className="text-2xl">
                     {/* Total Positive Cases Since {props.showAllTime ? "August 19 (First Day of Classes)" : "January 1"} */}
-                    Total Positive Cases Since August 16 (First Day of Move-In)
+                    Total Positive Cases Since January 11 (Second Day of Classes)
```

Comment:
It's technically since Jan 10, the first day of classes, I just didn't get data for that day.